install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -0,0 +1,387 @@
+#!/bin/sh
+
+set -e
+
+TMP_DIR=/tmp/migrate-k8s
+LOG_LEVEL=info
+PRIVATE_ADDRESS=
+AIRGAP=0
+# If all state needed from the native app has been saved we can skip some steps
+HAS_NATIVE_STATE=0
+HAS_KUBERNETES=0
+HAS_APP=0
+AIRGAP_LICENSE_PATH=
+AIRGAP_PACKAGE_PATH=
+INIT_FLAGS="no-clear"
+
+{% include 'common/cli-script.sh' %}
+{% include 'common/common.sh' %}
+{% include 'common/kubernetes.sh' %}
+{% include 'common/log.sh' %}
+{% include 'common/system.sh' %}
+
+startNativeScheduler() {
+    logStep "Ensuring native scheduler is running"
+
+    # remove NodePort services since they will contend with native container published ports
+    if commandExists "kubectl" ; then
+        logSubstep "stop kubernetes services"
+        kubectl delete services replicated replicated-ui &>/dev/null || true
+    fi
+
+    logSubstep "start systemd units"
+    systemctl start replicated
+    systemctl start replicated-operator
+    systemctl start replicated-ui
+
+    # ensure replicatedctl wrapper is for native
+    logSubstep "configure replicatedctl for native scheduler"
+    if ! grep -q "sudo docker exec" "/usr/local/bin/replicatedctl" 2>/dev/null ; then
+        installCliFile "sudo docker exec" "replicated"
+    fi
+
+    logSubstep "wait for native daemon to report ready"
```

Comment:
how about moving the message into the `waitReplicatedctlReady` function?   that function is called from multiple places and not all of them have user feedback.

Reply:
moved